Javascript/home.js

```diff
@@ -0,0 +1,199 @@
+  {
+    name: 'Dr Kristel Bauer',
+    briefly:
+      'Keynote speaker and corporate wellness expert. She is the founder of be well',
+    mission:
+      'She is clinical psycologist who specialises in diagnosis and psychological treatment of mental, behavioral and emotional illnesses',
+    imageUrl: 'https://randomuser.me/api/portraits/women/1.jpg',
+  },
+  {
+    name: 'Dr Tali Sharot',
+    briefly:
+      'A neuroscientist provides keynote speeches on the science that empowers optimism using her expertise in the psychology and decision making.',
+    mission:
+      'Dr Tali Sharot is a visiting professor at MIT and also an associate professor at University College London',
+    imageUrl: 'https://randomuser.me/api/portraits/women/2.jpg',
+  },
+  {
+    name: 'Dr Martyn Newman',
+    briefly: 'Bestseller author, Mindfulness and Emotional Intelligence',
+    mission:
+      'Dr Martyn is a clinical Psychologist specialized in Mindfulness and emotional intelligence. As a best seller author of Emotional Capitalists',
+    imageUrl: 'https://randomuser.me/api/portraits/men/4.jpg',
+  },
+  {
+    name: 'Dr. Chloe Carmichael, PhD',
+    briefly: 'Chemical Psychologist, author and inspirational speaker',
+    mission:
+      'Dr Chloe is a leading expert on anxiety nationally know speaker, clinical psychologist, author and entrepreneur',
+    imageUrl: 'https://randomuser.me/api/portraits/women/8.jpg',
+  },
+  {
+    name: 'Elliot Aronson',
+    briefly:
+      'Aron is currently Professor Emeritus of psybhology at the university of California, Santa Cruz',
+    mission:
+      "Aronson's work has focussed on the social dimension of human motivation MediaKeySystemAccess, such as the structural reasons for prejudice and aggression",
+    imageUrl: 'https://randomuser.me/api/portraits/men/20.jpg',
+  },
+  {
+    name: 'Hayley Mulenda',
+    briefly: 'Passionate on yourth mental health activist',
+    mission:
+      'Recognised as the most influential and most inpiratinal black entreprener in Britain, Author of the award winning book ABCs to student success',
+    imageUrl: 'https://randomuser.me/api/portraits/women/10.jpg',
+  },
+  {
+    name: 'Nell Pasricha',
+    briefly:
+      'The new York times best seller of five books and one of the most popular TED speakers in the world',
+    mission:
+      "'The 3 A's of Awesome' continue to define resillience through mental health challenges for millions of people",
+    imageUrl: 'https://randomuser.me/api/portraits/men/13.jpg',
+  },
+  {
+    name: 'Mellissa Doman, M.A',
+    briefly:
+      'Organisational Pshycologist, Mental at Work Specialist and Former Mental Health Therapist',
+    mission:
+      'Melissa Doma uses her years as a mental health professional to assist variety of organisations in implementing and becoming aware of the importace of emotinal and mental welness in a work place',
+    imageUrl: 'https://randomuser.me/api/portraits/women/5.jpg',
+  },
+];
```

Comment:
- [ ] Requirements are to have 6 featured speakers articles, but you have 8. It's a great thing that this section of your code is generated dynamically so you just have to remove 2 items from this array and you would have met this requirement 👍